index.css

```diff
@@ -40,7 +41,17 @@ a:hover, a:active {
   text-decoration: underline;
 }
 
+#results {
+  overflow: auto;
+  height: 430px;
```

Comment:
Probably having a fixed height here is not a good idea, I think. Can try `height: calc(100% - 20px);`.